packages/config/tsconfig/vite.json

```diff
@@ -8,8 +8,8 @@
     "sourceMap": true,
     "resolveJsonModule": true,
     "noEmit": true,
-    "noUnusedLocals": true,
-    "noUnusedParameters": true,
+    "noUnusedLocals": false,
+    "noUnusedParameters": false,
```

Comment:
imo it's not typescripts job to warn about unused stuff - that's up to the linter